### PR TITLE
Mark meetings as done/undone, and show only un-done meetings

### LINKED ISSUE #111: Mark meetings as done
Currently, there is no way to mark meetings as done and the meetings are just clogging up the list even though they are over/done.

With this, there should also be a way to show all meetings that are done

### PR DESCRIPTION
**Pull request by chuakid**
1. `mark` command for marking meetings as done
2. `unmark` for doing the opposite
3. `pending` command shows all meetings that are either in the future or not marked as done

Resolves #111

### CODE REVIEW

File: src/main/java/seedu/quickcontacts/logic/commands/ShowNotDoneCommand.java
Comment (Beebeeoii) [inline src/main/java/seedu/quickcontacts/logic/commands/ShowNotDoneCommand.java:1]: haha could we rename the commandword to something like `pending`? `undone` seems like marking something as undone

File: src/main/java/seedu/quickcontacts/model/meeting/Meeting.java [hunk 1]
```diff
@@ -23,8 +23,7 @@ public class Meeting {
     // Data fields
     private final Location location;
     private final Description description;
-
-    private boolean hasPassed = false;
+    private final boolean isDone;
```
Comment (Beebeeoii) [inline src/main/java/seedu/quickcontacts/model/meeting/Meeting.java:26]: minor nit but for english's sake, could we rename this to `isCompleted`

**Review by Beebeeoii — APPROVED** [review]
lgtm ! thanks for this !

File: src/main/java/seedu/quickcontacts/logic/commands/ShowNotDoneCommand.java
Comment (chuakid) [reply src/main/java/seedu/quickcontacts/logic/commands/ShowNotDoneCommand.java:1]: Resolved with 69583fd

File: src/main/java/seedu/quickcontacts/model/meeting/Meeting.java (hunk 1 above)
Comment (chuakid) [reply src/main/java/seedu/quickcontacts/model/meeting/Meeting.java]: Resolved with 69583fd

File: src/main/java/seedu/quickcontacts/logic/commands/MarkAsDoneCommand.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+/**
+ * Creates a new command that marks a meeting as done
+ */
+public class MarkAsDoneCommand extends Command {
+    public static final String COMMAND_WORD = "mark";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Marks a meeting as done\n" + "Parameters: m/ "
+            + "index of meeting";
+    public static final String INDEX_NOT_FOUND = "One of the provided indices is not found";
+    public static final String COMMAND_DESCRIPTION = "Marks a meeting as done";
```
Comment (KS2499) [inline src/main/java/seedu/quickcontacts/logic/commands/MarkAsDoneCommand.java:19]:
```suggestion
    public static final String COMMAND_DESCRIPTION = "Marks a meeting as done.";
```

**Review by KS2499 — APPROVED** [review]
LGTM! Some very minor punctuation, otherwise solid work!

**Pull request merged**